src/main/java/org/gelecekbilimde/scienceplatform/common/model/response/Response.java

```diff
@@ -26,15 +26,15 @@ public class Response<T> {
 	public static <T> Response<T> ok(final T response) {
 		return Response.<T>builder()
 			.list(response)
-			.responseCode(UUID.randomUUID().toString())
+			.responseCode(RandomUtil.generateUUID())
```

Comment:
Burada Javanın UUID class'ı yerine neden kendi class'ımı yazıp kullandık?

Reply:
Kendi classımınz değil ki o javanın içinden gelen bir UUID sınıfı

Reply:
İleride kendi özel UUID özelleştirmemiz için olabilir. Mesela gb-123214123-123123

Reply:
Evet o şekilde güncellemek daha kolay olabilir, bir de uygulama genelinde var olduğu için komple düzeltmek yerine kullanalım istedim